cmd/setup/integration_test/setup_test.go

```diff
@@ -11,14 +11,30 @@ package setup_test
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"testing"
 	"time"
 
 	"github.com/jackc/pgx/v5/pgxpool"
 )
 
-const ConnString = "host=localhost port=5432 user=zitadel dbname=zitadel sslmode=disable"
+func getEnv(key, fallback string) string {
+	if value, exists := os.LookupEnv(key); exists {
+		return value
+	}
+	return fallback
+}
+
+var ConnString = fmt.Sprintf(
+	"host=%s port=%s user=%s password=%s dbname=%s sslmode=%s",
+	getEnv("ZITADEL_DATABASE_POSTGRES_HOST", "localhost"),
+	getEnv("ZITADEL_DATABASE_POSTGRES_PORT", "5433"),
+	getEnv("ZITADEL_DATABASE_POSTGRES_USER", "zitadel"),
+	getEnv("ZITADEL_DATABASE_POSTGRES_PASSWORD", "zitadel"),
+	getEnv("ZITADEL_DATABASE_POSTGRES_DATABASE", "zitadel"),
+	getEnv("ZITADEL_DATABASE_POSTGRES_SSL_MODE", "disable"),
+)
```

Comment:
@adlerhurst are you fine with these changes?
I added them mainly because we need to address the DB using the compose service name when we are in the dev container. So we can't just use the hardcoded `localhost` for the DB host.

Don't get confused about the default port, it's 5433 on purpose for integration tests now.